63931 file travel question content

lee-delarm6 opened the pull request:
## Summary

- Update travel question text and layout slightly

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/63931#issuecomment-1697396611

## Testing done

- Screenshots
- Testing on local

## Screenshots
<img width="1072" alt="Screenshot 2023-08-29 at 2 18 36 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/135059743/8ee0275c-d46f-4827-bad7-65c5eabecda1">
<img width="1054" alt="Screenshot 2023-08-29 at 2 18 41 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/135059743/070ee792-b9fc-430a-b3e9-fa588b98fd35">


## What areas of the site does it impact?

Travel question text and layout

## Acceptance criteria

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added

lee-delarm6 committed:
Update Travel question

Updated travel question (although not final)

lee-delarm6 committed:
Travel question update

Travel question update

lee-delarm6 committed:
Wording update

Wording update

brianseek commented on src/applications/check-in/locales/en/translation.json:
```diff
@@ -191,7 +191,14 @@
+  "youre-traveling-for-treatment-of-a-service-connected-condition": "You're traveling for treatment of a service-connected condition.",
+  "you-receive-va-pension-benefits": "You receive VA pension benefits.",
+  "you-have-an-annual-income-below-the-maximum": "You have an annual income below the maximum annual VA pension rate.",
+  "youre-traveling-in-relation-to-a-compensation-and-pension": "You're traveling to a Compensation and Pension, or C&P, exam.",
```
when adding new translations, put them at the end of the file until we get the translations back from the state dept.

brianseek commented on src/applications/check-in/day-of/pages/TravelQuestion.jsx:
```diff
@@ -16,20 +67,19 @@ const TravelQuestion = props => {
-      pageType="travel-question"
-      router={router}
-    />
+    <>
```
not a big deal, but why the addition of the empty tags?

lee-delarm6 replied:
This is a mistake, I think initially I had the idea I was going to add more elements into this area and ended up not doing that

brianseek commented on src/applications/check-in/components/pages/TravelPage/index.jsx:
```diff
@@ -19,6 +19,7 @@ const TravelPage = ({
   eyebrow,
   bodyText,
   helpText,
+  additionalInfo,
```
Add to propTypes

brianseek reviewed (changes requested):
Couple little changes and question.

lee-delarm6 committed:
Minor fixes and changes

Added proptype, moved translations, removed extra empty brackets, added small test for travel page check for new prop

brianseek reviewed (approved):
Updates look great 🥇

lee-delarm6 committed:
Merge branch 'main' into 63931_file_travel_question_content

lee-delarm6 committed:
Merge branch 'main' into 63931_file_travel_question_content

lee-delarm6 committed:
Merge branch 'main' into 63931_file_travel_question_content

lee-delarm6 committed:
Add missing comma from merge

Add missing comma from merge

lee-delarm6 committed:
Merge branch 'main' into 63931_file_travel_question_content

lee-delarm6 committed:
Merge branch 'main' into 63931_file_travel_question_content

lee-delarm6 committed:
Merge branch 'main' into 63931_file_travel_question_content

lee-delarm6 committed:
Merge branch 'main' into 63931_file_travel_question_content

lee-delarm6 committed:
Merge branch 'main' into 63931_file_travel_question_content
